public/app/percona/integrated-alerting/components/Alerts/Alerts.tsx

```diff
@@ -169,7 +169,7 @@ export const Alerts: FC = () => {
       setPageSize(pageSize);
       setPageindex(pageIndex);
     },
-    [setPageSize]
+    [setPageSize, setPageindex]
```

Comment:
can we change to CamelCase? setPageIndex?

Reply:
Well spotted :D